platform/ios/src/MGLMapView.h

```diff
@@ -952,6 +952,15 @@ IB_DESIGNABLE
 - (void)removeAnnotations:(NS_ARRAY_OF(id <MGLAnnotation>) *)annotations;
 
 /**
+ Returns an MGLAnnotationView if the given annotation is currently associated
+ with a view, otherwise nil.
```

Comment:
Nit: put MGLAnnotationView in backticks so jazzy knows to linkify it. (Xcode renders the backticks, unfortunately, but I think the jazzy links are worth it.)